quinn-proto/src/shared.rs

```diff
@@ -257,33 +273,81 @@ pub struct ServerConfig<S>
 where
     S: crypto::Session,
 {
-    /// Transport configuration to use for incoming connections
+    /// TLS configuration used for incoming connections.
```

Comment:
Not sure why this change -- `TransportConfig` does not contain TLS configuration.